.github/workflows/upload-content-files.yml

```diff
@@ -77,3 +111,41 @@ jobs:
         uses: actions/checkout@v2
       - run: git pull -r
       - run: aws s3 cp ./config/forms/forms.json "s3://$BUCKET/forms.json"
+
+  push-answer-filters-to-production:
```

Comment:
As answer filters are updated when forms change, I would combine them together.

Reply:
👍🏾 Going to check this again with Elsa.

Reply:
Just saw Jaye's reply. Yep, will do.